blackbox/docs/appendices/release-notes/4.0.5.rst

```diff
@@ -0,0 +1,75 @@
+.. _version_4.0.5:
+
+=============
+Version 4.0.5
+=============
+
+Released on 2019/09/19.
+
+.. NOTE::
+
+    Please consult the :ref:`version_4.0.0_upgrade_notes` before upgrading from
+    CrateDB 3.x or earlier.
+    Before upgrading to 4.0.5 you should be running a CrateDB cluster that is
+    at least on 3.0.7.
+
+    We recommend that you upgrade to the latest 3.3 release before moving to 4.0.5.
+
+    If you want to perform a `rolling upgrade`_, your current CrateDB version
+    number must be at least :ref:`version_4.0.2`. Any upgrade from a version
+    prior to this will require a `full restart upgrade`_.
+
+    When restarting, CrateDB will migrate indexes to a newer format. Depending
+    on the amount of data, this may delay node start-up time.
+
+.. WARNING::
+
+    Tables that were created prior CrateDB 3.x will not function with 4.x
+    and must be recreated before moving to 4.x.x.
+
+    You can recreate tables using ``COPY TO`` and ``COPY FROM`` or by
+    `inserting the data into a new table`_.
+
+    Before upgrading, you should `back up your data`_.
+
+.. _rolling upgrade: http://crate.io/docs/crate/guide/best_practices/rolling_upgrade.html
+.. _full restart upgrade: http://crate.io/docs/crate/guide/best_practices/full_restart_upgrade.html
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+.. _inserting the data into a new table: https://crate.io/docs/crate/reference/en/latest/admin/system-information.html#tables-need-to-be-recreated
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+
+See the :ref:`version_4.0.0` release notes for a full list of changes in the
+4.0 series.
+
+Changes
+=======
+
+- Improved the help section of the admin-ui and added Spanish translations.
```

Comment:
The hotfix should probably contain only fixes? Also, can we maybe rephrase a bit the part about the Spanish translation, I had to check the Admin-UI release notes to understand what was actually translated/fixed. https://github.com/crate/crate-admin/commit/7c7763ba9c4bf83d2bf1720a1579e6ea4cb1692a#diff-bf59ca62b9676f8b8e05f808bdd46628L132

Reply:
yes this is a bit of a grey area as the related commit contains fixes and improvements. I'd suggest to move this entry also to the `Fixes` and rephrase it to `improved Spanish translations`.

Reply:
> yes this is a bit of a grey area as the related commit contains fixes and improvements. I'd suggest to move this entry also to the `Fixes` and rephrase it to `improved Spanish translations`.

Ok, do we just do this in this release pr or also in the admin-ui commit before ?